### PR TITLE
Plugins docs

### PR DESCRIPTION
**Pull request by denCalderon**
Documentation regarding 3rd-party plugins.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ utilize the latest Intel hardware features.
   - [Known Issues](#known-issues)
 - [Contributing](#contributing)
   - [Adding a New Command](#adding-a-new-command)
+  - [Third Party Plugins](#third-party-plugins)
```
Comment (hamishun) [inline README.md:38]: This is a feature. Should appear before Contributuions in its own section.

File: README.md [hunk 2]
```diff
@@ -309,6 +310,11 @@ described the functionality of the new command.
 
 * Generic utilities or helper functions that can be used for several commands
   should be in [utils](kit/utils).
+  
+## Third Party Plugins
```
Comment (hamishun) [inline README.md:314]: This is a feature. Should appear before Contributuions in its own section.

**Review by hamishun — CHANGES_REQUESTED** [review]
format tests are failing.

File: kit/PLUGINS.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    )
+    parser_TBD.add_argument(
+        "ARG2", description="ADD-ARG-DESCRIPTION"
+    )
+    parser_TBD.set_defaults(fn=NEW_FUNCTIONALITY)
+```
+
+The parameter `fn` of the function `set_defaults` must be set to the
+function that implements the entry point of the plugin, therefore, it
+uses the arguments defined in the previous step.
```
Comment (jlhcrawford) [inline kit/PLUGINS.md:139]: I am not sure I understand what this paragraph means.

File: kit/PLUGINS.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    """Executes new functionality"""
+    if(args.ARG1)
+        pass
+    elif(args.ARG2)
+        pass
+```
+
+### Main directory
+The plugin root directory must have the same name as the plugin, thus
+it must be the same as the `name` value defined in the TOML file. This
```
Comment (jlhcrawford) [inline kit/PLUGINS.md:151]: Is this true? The name of the directory must match the plugin name?
If so the example you provided above doesn't follow this rule.

**Review by hamishun — APPROVED** [review]

**Review by jlhcrawford — APPROVED** [review]

**Pull request merged**